### DISCUSSION TITLE
You are not allowed to perform this action when ran in production

### DISCUSSION THREAD

**Discussion by BigHoss** [payloadcms/payload#8651] [Q&A]
# Steps to Recreate
1. create a 3.0 beta website on my local machine
1. connect to the production postgres database
1. seed the database
1. activate localization an l18n
1. edit pages and content in debug
1. push changes into repo
1. coolify builds and deploys payload
1. when editing anything in the admin interface, I get errors and strange behavior
  1. error log messages 
  1. validation errors in ui and ui breaking

Am I doing something wrong here or how am I supposed to do this properly?

# errors in log
<img width="1451" alt="log error messages" src="https://github.com/user-attachments/assets/cb50762a-d7f9-41cd-ab04-df200a12c6b1">

# validation errors when publishing a page
<img width="957" alt="publishing a page" src="https://github.com/user-attachments/assets/e4fdb874-3715-4556-85c1-30cd32deabcf">

# breaking ui when switching locale
<img width="876" alt="changeing locale on the same page" src="https://github.com/user-attachments/assets/649cc692-7bbf-447d-b906-79a397c5caa3">

**Comment by BigHoss** [comment]
Now i dropped all the tables and let the published installation do the migration and reregister. Still the same issue.

Can please someone help me with this?

**Reply by fazofazaal** [reply]
I'm also having the same issue.

The prod site is on Vercel. I've connected to a Mongo DB Atlas instance on both local  and live sites. 

Can create, update, and delete resources locally, but getting the same errors on the live site.

**Reply by JP-HoneyBadger** [reply]
I am seeing something similar to this related to Auth.  If you change create and update to anyone in the collection, does it work?

**Reply by fazofazaal** [reply]
Yeah, it does work. However, there is still a 401 unauthorized error response from the form_state endpoint, which is used by the Lexical editor to update its state. The media blocks within Lexical editor don't work in production

**Reply by sekiranat** [reply]
> Yeah, it does work. However, there is still a 401 unauthorized error response from the form_state endpoint, which is used by the Lexical editor to update its state. The media blocks within Lexical editor don't work in production

Try use blank template

Edit: The template has no effect on this issue

**Comment by sekiranat** [comment]
This error is only reproduced for the template website and is due to `user === null` during request validation in `src/access/authenticated.ts`

Edit: The template has no effect on this issue

**Comment by exoer** [comment]
Check if there is a mismatch between the env-variable PAYLOAD_PUBLIC_SERVER_URL and the actual domain you are running on

**Reply by sekiranat** [reply]
Yes, the problem was that localhost:3000 was busy and the panel automatically opened on port 3001. Env is set to port 3000 by default.
